Src/AutoFixture/RegularExpressionGenerator.cs

```diff
@@ -10,6 +10,16 @@ namespace AutoFixture
     /// </summary>
     public class RegularExpressionGenerator : ISpecimenBuilder
     {
+        private readonly Random random;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RegularExpressionGenerator"/> class.
+        /// </summary>
+        public RegularExpressionGenerator()
+        {
+            this.random = new Random();
```

Comment:
This looks reasonable *if* RegularExpressionGenerator has also the same, transient, scope.

(A test similar to [this](https://github.com/moodmosaic/Fare/pull/56/commits/1563f8e5cea71bb58c85e7843498796d81975715#diff-a4691edefa1cec0b62875a993b25ed7c) one, which you did previously, should give the answer.)

Reply:
ok - I've added a test in FixtureTests.